src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -0,0 +1,448 @@
+                    }
+                }
+
+                Model model = null;
+                using (StreamReader sr = File.OpenText(_modelFileNameWithPath))
+                {
+                    string line;
+                    int lineNumber = 1;
+                    char[] delimiters = { ' ', '\t' };
+                    using (var ch = Host.Start(LoaderSignature))
+                    using (var pch = Host.StartProgressChannel("Building Vocabulary from Model File for Word Embeddings Transform"))
+                    {
+                        var header = new ProgressHeader(new[] { "lines" });
+                        pch.SetHeader(header, e => e.SetProgress(0, lineNumber));
+                        string firstLine = sr.ReadLine();
+                        while ((line = sr.ReadLine()) != null)
+                        {
+                            if (lineNumber >= _linesToSkip)
+                            {
+                                string[] words = line.TrimEnd().Split(delimiters);
+                                dimension = words.Length - 1;
+                                if (model == null)
+                                    model = new Model(dimension);
+                                if (model.Dimension != dimension)
+                                    ch.Warning($"Dimension mismatch while reading model file: '{_modelFileNameWithPath}', line number {lineNumber + 1}, expected dimension = {model.Dimension}, received dimension = {dimension}");
+                                else
+                                {
+                                    float tmp;
+                                    string key = words[0];
+                                    float[] value = words.Skip(1).Select(x => float.TryParse(x, out tmp) ? tmp : Single.NaN).ToArray();
+                                    if (!value.Contains(Single.NaN))
+                                        model.AddWordVector(ch, key, value);
+                                    else
+                                        ch.Warning($"Parsing error while reading model file: '{_modelFileNameWithPath}', line number {lineNumber + 1}");
+                                }
+                            }
+                            lineNumber++;
+                        }
+
+                        // Handle first line of the embedding file separately since some embedding files including fastText have a single-line header 
+                        string[] wordsInFirstLine = firstLine.TrimEnd().Split(delimiters);
+                        dimension = wordsInFirstLine.Length - 1;
+                        if (model == null)
+                            model = new Model(dimension);
+                        if (model.Dimension != dimension)
+                            ch.Warning($"Dimension mismatch while reading model file: '{_modelFileNameWithPath}', line number 1, expected dimension = {model.Dimension}, received dimension = {dimension}");
```

Comment:
We can remove this warning. The purpose of this block of code is to allow the 1st line to be of different length (and ignored if so).  Hence the warning is superfluous. Currently this warning is displayed for all fastText models, even though the model is read correctly (by ignoring the 1st line). 

Background: In fastText models, the 1st line is: <fileLength> <embeddingWidth>. Other word embedding models don't have a header line.